README.md

```diff
@@ -1333,6 +1430,104 @@ If the root object in the body is missing both the "variable" and the
 
 Upon completion, 200 is returned along with the full table definition.
 
+### https://[APPURL]/api/Table/[id]/filter
+
+<h4 id="table_id_filter_get">GET</h4>
+
+Return the api search preferences for a specific table (with a given id). <br> 
+Returns the api search preferences at the Table("self") level.
+Below is an example result body. 
+```json
+{
+    "self": {
+        "api_preference": {
+            "ex_api": [
+                "ontology1",
+                "onto2"
+            ]
+        }
+    }
+}
+```
+
+#### PUT
+
+Create a api search preference for a specific table (with a given id). <br> 
+See the [POST request](#table_id_filter_post) an example request body for this endpoint.
+
+<h4 id="table_id_filter_post">POST</h4>
+
+Update the api search preferences for a specific table (with a given id). <br>
+Request body example:
+ ```json
+{
+    "api_preference": {
+        "ex_api": ["ontology1", "onto2"]
+    }
+}
+```
+
+#### DELETE
+
+Delete the api search preferences for a specific table (with a given id). An example of the data after removal shown below.
+```json
+{}
+```
+
+### https://[APPURL]/api/Table/[id]/filter/[code]
+
+<h4 id="table_id_filter_code_get">GET</h4>
```

Comment:
Those "#" characters are this flavor of Markdown's header indication. So, H4 is four pound signs: "#### GET" would be the same as your h4 tag above, unless you need that id to be specific.